src/Security/HybrideDigitalAuthenticator.php

```diff
@@ -100,9 +100,6 @@ public function onAuthenticationSuccess(Request $request, TokenInterface $token,
         if ($targetPath) {
             return new RedirectResponse($targetPath);
         }
-
-        return new RedirectResponse($this->urlGenerator->generate('app_register'));
```

Comment:
Why delete this return ?? You need it if people comes directly from /login

Reply:
sorry it was for a test , I resolve it